### DISCUSSION TITLE
New Features

### DISCUSSION THREAD

**Discussion by Cahllagerfeld** [EddieHubCommunity/good-first-issue-finder#185] [Ideas]
We're planning to add a new set of new Features. These are the following ideas: 
- Turning Authentication to Github OAuth so users can log in with their own account
- Displaying Version Number
- Adding Detail Page for Issues
- Display more information about Repo itself
- Add Pagination / Infinite Scroll

**Comment by Cahllagerfeld** [comment]
We're currently using a Personal-Access-Token to request the Github API. That's not a very clean way, although working. We could migrate it to use Oauth

**Reply by eddiejaoude** [reply]
This would be great 👍

**Reply by schmelto** [reply]
https://docs.github.com/en/developers/apps/building-oauth-apps/authorizing-oauth-apps this would be the documentation for this ig

**Reply by Cahllagerfeld** [reply]
closed with #187

**Comment by Cahllagerfeld** [comment]
Display of Version Number to show the current version the project has in the UI

**Reply by Cahllagerfeld** [reply]
closed with #197 and #198

**Comment by Cahllagerfeld** [comment]
Adding Detail Pages, so when an issue gets clicked, the app doesn't route to Github but displays the most relevant data about repository and issue in the app itself

**Comment by Cahllagerfeld** [comment]
Add Pagination, so we can display more issues. This was currently not needed, because there weren't toom any issues to display, but might be a thing for the future

**Reply by Cahllagerfeld** [reply]
tackled with #199
